Document using setup.cfg for src-layouts (#1537)
pypa/setuptools: docs/setuptools.txt

* Unknown keys are ignored.

> Using a ``src/`` layout (storing packages in a subdirectory)

Reviewer: ```suggestion
Using a ``src/`` layout
```

I think maybe we don't need the parenthetical part of this.

============================================================

> If you use a ``src/`` layout, as in, you package structure looks like this::

Reviewer: ```suggestion
One commonly used package configuration has all the module source code in a subdirectory (often called the `src/` layout), like this::
```

    ├── src
    │   └── mypackage
    │       ├── __init__.py
    │       └── mod1.py
    ├── setup.py
    └── setup.cfg

> You can set up your ``setup.cfg`` to automatically look up all your packages in

Reviewer: ```suggestion
You can set up your ``setup.cfg`` to automatically find all your packages in
```
